### PR TITLE
Feature/corona sample rework

### PR DESCRIPTION
**Pull request by shanerudolfworktive**

### CODE REVIEW

File: android-samples/SimpleSample/src/main/java/me/kiip/simplesample/SampleApplication.java [hunk 1]
```diff
@@ -17,8 +17,8 @@
 public class SampleApplication extends Application implements OnContentListener {
 
     public static final String TAG = "kiip";
-    private static final String APP_KEY = "my_app_key";
-    private static final String APP_SECRET = "my_app_secret";
+    private static final String APP_KEY = "e034e855dee6bc37a2d1a0077719e358";
```
Comment (sraj) [inline android-samples/SimpleSample/src/main/java/me/kiip/simplesample/SampleApplication.java]: Don't check-in

Comment (shanerudolfworktive) [inline android-samples/SimpleSample/src/main/java/me/kiip/simplesample/SampleApplication.java]: Oops, I revert it back

File: corona-samples/SpaceShooter/ios/AppCoronaDelegate.mm [hunk 2]
```diff
@@ -9,29 +9,29 @@
 
 #import "CoronaRuntime.h"
 #import "CoronaLua.h"
-#import <KiipSDK/KiipSDK.h>
+//#import <KiipSDK/KiipSDK.h>
```
Comment (sraj) [inline corona-samples/SpaceShooter/ios/AppCoronaDelegate.mm]: If it's not required remove it

File: corona-samples/SpaceShooter/ios/AppCoronaDelegate.mm [hunk 3]
```diff
@@ -9,29 +9,29 @@
...
+//                // handle with an Alert dialog.
+//            }
+//            if (poptart) {
+//                [poptart show];
+//            }
+//            if (!poptart) {
+//                // handle logic when there is no reward to give.
+//            }
+//        }];
+//    });
```
Comment (sraj) [inline corona-samples/SpaceShooter/ios/AppCoronaDelegate.mm]: If it's not required remove it

**Comment by sraj** [comment]
👍

**Pull request merged**